app/Providers/ModelServiceProvider.php

```diff
@@ -37,6 +39,22 @@ public function boot()
                 'content' => $review->toJson(),
             ]);
         });
+
+        Tagged::saved(function ($tagged) {
+            $post = Post::find($tagged->taggable_id);
+            $adminId = auth()->user()->northstar_id;
+
+            $post->action = [
+                'type' => 'tagged',
+                'admin' => $adminId,
+            ];
+
+            Event::create([
+                'eventable_id' => $post->id,
+                'eventable_type' => 'Conner\Tagging\Model\Tagged',
```

Comment:
Sorry I wonder if this `eventable_type` should be `Rogue\Models\Post` because we are logging it as an event on the post even tho it's the `Tagged` model that triggers it.